desc/grid.py

```diff
@@ -34,12 +34,16 @@ class _Grid(IOAble, ABC):
         "_weights",
         "_axis",
         "_node_pattern",
+        "_coordinates",
+        "_period",
+        "_source_grid",
         "_unique_rho_idx",
-        "_unique_theta_idx",
+        "_unique_poloidal_idx",
         "_unique_zeta_idx",
         "_inverse_rho_idx",
-        "_inverse_theta_idx",
+        "_inverse_poloidal_idx",
```

Comment:
should add logic to `_set_up` to handle these new/renamed attributes when loading old solutions